src/store/create-store.ts

```diff
@@ -0,0 +1,5 @@
+import { DatabaseOptions, Level } from 'level';
+
+export function createStore(location: string, options?: DatabaseOptions<string, Uint8Array>): Level<string, Uint8Array> {
```

Comment:
i was actually trying to make this more generic in case we decided to ditch `Level` while still having something named relevantly to `MessageStore` and `DataStore` (i.e. more immediately discernible what `createStore` would be used by)

Reply:
i do kinda like the idea of `createDatabase` tho 🤔